src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+
+    @Inject
+    UserDao userDao;
+
+    public void handleBet() {
+        System.out.println("Enter 'value' and 'risk' for your bet");
```

Comment:
I would add an example or description of the required format of input data for the user to avoid mistakes. 
A simple instruction :)
But it's up to you.